devicetypes/fibargroup/fibaro-dimmer-2-zw5.src/fibaro-dimmer-2-zw5.groovy

```diff
@@ -0,0 +1,498 @@
+                attributeState "off", label: 'Off', action: "on", icon: "https://s3-eu-west-1.amazonaws.com/fibaro-smartthings/dimmer/dimmer0.png", backgroundColor: "#ffffff", nextState:"turningOn"
+                attributeState "on", label: 'On', action: "off", icon: "https://s3-eu-west-1.amazonaws.com/fibaro-smartthings/dimmer/dimmer100.png", backgroundColor: "#00a0dc", nextState:"turningOff"
+                attributeState "turningOn", label:'Turning On', action:"off", icon:"https://s3-eu-west-1.amazonaws.com/fibaro-smartthings/dimmer/dimmer50.png", backgroundColor:"#00a0dc", nextState:"turningOff"
+                attributeState "turningOff", label:'Turning Off', action:"on", icon:"https://s3-eu-west-1.amazonaws.com/fibaro-smartthings/dimmer/dimmer50.png", backgroundColor:"#ffffff", nextState:"turningOn"
+            }
+            tileAttribute("device.multiStatus", key:"SECONDARY_CONTROL") {
+                attributeState("combinedMeter", label:'${currentValue}')
+            }
+            tileAttribute ("device.level", key: "SLIDER_CONTROL") {
+                attributeState "level", action:"switch level.setLevel"
+            }
+        }
+        valueTile("power", "device.power", decoration: "flat", width: 2, height: 2) {
+            state "power", label:'${currentValue}\nW', action:"refresh"
+        }
+        valueTile("energy", "device.energy", decoration: "flat", width: 2, height: 2) {
+            state "energy", label:'${currentValue}\nkWh', action:"refresh"
+        }
+        valueTile("reset", "device.energy", decoration: "flat", width: 2, height: 2) {
+            state "reset", label:'reset\nkWh', action:"reset"
+        }
+
+        valueTile("scene", "device.scene", decoration: "flat", width: 2, height: 2) {
+             state "default", label:'Scene: ${currentValue}'
+        }
+
+        standardTile("errorMode", "device.errorMode", decoration: "flat", width: 2, height: 2) {
+            state "default", label:'No errors.', action:"clearError", icon: "st.secondary.tools", backgroundColor: "#ffffff"
+            state "overheat", label:'Overheat!', action:"clearError", icon: "st.secondary.tools", backgroundColor: "#ff0000"
+            state "surge", label:'Surge!', action:"clearError", icon: "st.secondary.tools", backgroundColor: "#ff0000"
+            state "voltageDrop", label:'Voltage drop!', action:"clearError", icon: "st.secondary.tools", backgroundColor: "#ff0000"
+            state "overcurrent", label:'Overcurrent!', action:"clearError", icon: "st.secondary.tools", backgroundColor: "#ff0000"
+            state "overload", label:'Overload!', action:"clearError", icon: "st.secondary.tools", backgroundColor: "#ff0000"
+            state "loadError", label:'Load Error!', action:"clearError", icon: "st.secondary.tools", backgroundColor: "#ff0000"
+            state "hardware", label:'Hardware Error!', action:"clearError", icon: "st.secondary.tools", backgroundColor: "#ff0000"
+        }

```

Comment:
If for no other reason than clarity, this device should set the `main` and `details` tiles.